src/GraphQL.StarWars/GraphQL.StarWars.csproj

```diff
@@ -10,4 +10,20 @@
     <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="3.1.*" />
   </ItemGroup>
 
+  <ItemGroup>
+    <!-- 
+      The PackageReference to GraphQL.Analyzers is automatically included when referencing
```

Comment:
I dunno.  ChatGPT only gets you halfway there lol

Reply:
Alternatively, it can be added to `Directory.Build.props` with a condition:

```xml
<ItemGroup Condition="'$(UseAnalyzers)' == 'true'">
  <ProjectReference Include="..\GraphQL.Analyzers\GraphQL.Analyzers.csproj"
                    ReferenceOutputAssembly="false"
                    OutputItemType="Analyzer" />
  <ProjectReference Include="..\GraphQL.Analyzers.CodeFixes\GraphQL.Analyzers.CodeFixes.csproj"
                    ReferenceOutputAssembly="false"
                    OutputItemType="Analyzer" />
</ItemGroup>
```

So here, it will be only
```xml
<UseAnalyzers>true</UseAnalyzers>
```